docs/concepts/storage.md

```diff
@@ -0,0 +1,97 @@
+
+Someday this value will probably change, and when it does this document will be updated to include information about how to query the live version.
+
+## Example cost breakdown
+
+Let's walk through an example.
+
+A non-fungible token is one where each token is unique, which means each token has an ID. Amongst other things, such a contract must store a mapping from token IDs to owners' account addresses.
+
+If such an NFT is used to track **1 million** tokens, how much storage will be required, just for that token-ID-to-owner mapping? And how many tokens will need to be staked for that storage?
+
+Using [this basic AssemblyScript implementation](https://github.com/near-examples/NFT/tree/master/contracts/assemblyscript/nep4-basic) as inspiration, let's calculate the storage needs when using a [`PersistentMap`](https://near.github.io/near-sdk-as/classes/_assembly_sdk_collections_persistentmap_.persistentmap.html) from `near-sdk-as`. While the specific implementation may change in the future, at the time of writing it stored data as UTF-8 strings; let's assume that for this storage breakdown. Here's our `PersistentMap`:
+
+```ts
+type AccountId = string
+type TokenId = u64
+const tokenToOwner = new PersistentMap<TokenId, AccountId>('t2o')
+```
+
+Behind the scenes, all data stored on the NEAR blockchain is saved in a key-value database. That `'t2o'` variable that's passed to `PersistentMap` helps it keep track of all its values. If your account `mytokeniscoolerthanyours.near` owns token with ID `0`, then at the time of writing, here's the data that would get saved to the key-value database:
+
+* key: `t2o::0`
+* value: `mytokeniscoolerthanyours.near`
+
+So for 1 million tokens, here are all the things we need to add up and multiply by 1 million:
+
+1. The prefix, `t2o`, will be serialized as three bytes in UTF-8, and the two colons will add another two. That's 5 bytes.
+2. For an implementation where `TokenId` auto-increments, the values will be between `0` and `999999`, which makes the average length 5 bytes.
+3. Let's assume well-formed NEAR `AccountId`s, and let's guess that NEAR Account IDs follow the approximate pattern of domain names, which [average about 10 characters](https://www.domainregistration.com.au/news/2013/1301-domain-length.php), plus a top-level name like `.near`. So a reasonable average to expect might be about 15 characters; let's keep our estimate pessimistic and say 25. This will equal 25 bytes, since NEAR account IDs must use characters from the ASCII set.
+
+So:
+
+    1_000_000 * (5 + 5 + 25)
+
+35 million bytes. Multiplying by 9.09E19 yoctoNEAR per byte, we find that the `tokenToOwner` mapping will require staking 3.1815E27 yoctoNEAR, or Ⓝ3,181.5
+
+Note that you can get this down to Ⓝ2,999.7 just by changing the prefix from `t2o` to a single character
```

Comment:
+100 to providing suggestions on how to minimize costs! Are there any other tips we can come up with at this point?

Reply:
There's a whole section on this at the bottom 😄

Reply:
...which I now realize I failed to push before you reviewed! Sorry; I had it written but failed to push 😞